spectrum/autism_prevalence_map/templates/autism_prevalence_map/about.html

```diff
@@ -34,6 +34,138 @@
+                Research cohort — individuals identified for research projects and followed over time</p>
+                <p><b>Category</b> includes the following options:<br>
+                AD — autistic disorder<br>
+                PDD — pervasive developmental disorder<br>
+                ASD — autism spectrum disorder</p>
+                <div id="3" class="chapter-start"></div>
+                <h2 id="how-to-cite">How to cite the map:</h2>
+                <p>Please include a statement in your article or on your website that indicates that you retrieved the information from <a href="https://prevalence.spectrumnews.org">prevalence.spectrumnews.org</a>. And if you refer to it by name, please call it the “global map of autism prevalence by <em>Spectrum</em>.”</p>
+                <div id="4" class="chapter-start"></div>
+                <h2 id="more-info">More information:</h2>
+                <p>Here is a selection of <i>Spectrum</i> articles on autism prevalence.</p>
+                <p><a href="https://www.spectrumnews.org/news/study-quantifies-prevalence-autism-india/">Study quantifies prevalence of autism in India </a><br>
+                <a href="https://www.spectrumnews.org/news/new-report-shows-slight-uptick-autism-prevalence/">New report shows slight uptick in autism prevalence</a><br>
+                <a href="https://www.spectrumnews.org/news/prevalence-autism-u-s-remains-steady-new-data-suggest/">Prevalence of autism in U.S. remains steady, new data suggest</a><br>
+                <a href="https://www.spectrumnews.org/news/school-survey-india-reveals-low-autism-prevalence/">School survey in India reveals low autism prevalence</a><br>
+                <a href="https://www.spectrumnews.org/news/u-s-stats-show-autism-rate-reaching-possible-plateau/">U.S. stats show autism rate reaching possible plateau</a><br>
+                <a href="https://www.spectrumnews.org/news/new-spike-in-autism-numbers-may-reflect-rise-in-awareness/">New spike in autism numbers may reflect rise in awareness</a><br>
+                <a href="https://www.spectrumnews.org/opinion/prevalence-perturbed/">Prevalence perturbed</a><br>
+                <a href="https://www.spectrumnews.org/news/clinical-research-israeli-study-shows-high-autism-prevalence/">Clinical research: Israeli study shows high autism prevalence</a><br>
+                <a href="https://www.spectrumnews.org/opinion/rising-rates/">Rising rates</a><br>
+                <a href="https://www.spectrumnews.org/news/study-finds-high-rate-of-autism-in-south-korea/">Study finds high rate of autism in South Korea</a></p>
+                <div id="5" class="chapter-start"></div>
+                <h2 id="team">The team:</h2>
+                <h3><em>Spectrum</em> Team</h3>
+                <div class="bio">
+                    <h4>Apoorva Mandavilli</h4>
+                    <h5>Editor-in-Chief, <i>Spectrum</i></h5>
```

Comment:
use `<em>` instead of `<i>`. em and strong are semantic, while b and i are stylistic, so em and strong are preferred for accessibility